types/parcel/index.d.cts

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import {
+  ParcelConfig,
+  ParcelProps,
+  Parcel as SingleSpaParcel,
+} from "single-spa";
+
+interface ParcelCompProps<ExtraProps = {}> {
```

Comment:
Could this be moved to a shared declarations file, and then imported into both the .d.ts and .d.cts files?

Reply:
If you agree, we can leave this as-is for now, since these files will not be needed when we migrate to typescript.